metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -272,61 +278,15 @@ void convert_batch_to_row_diff(const std::string &graph_fname,
     std::vector<std::unique_ptr<annot::ColumnCompressed<>>> sources;
     for (const auto &fname : source_files) {
         auto anno = std::make_unique<annot::ColumnCompressed<>>() ;
-        anno->merge_load({fname});
+        anno->load(fname);
         sources.push_back(std::move(anno));
     }
     logger->trace("Done loading {} annotations", sources.size());
 
-    sdsl::rrr_vector rterminal;
-
-    // if we just generated anchor nodes, attempt a greedy anchor optimization
```

Comment:
It may achieve some reduction, it may also achieve some increase, which I don't like. So I'd stick to the new thing. Otherwise, we're gambling.
Besides, in order to check if that heuristic resulted in reduction or increase, you have to build another row-diff without this heuristic, so make the second run. And with the new approach we always have to run twice bug get an optimal result for sure :)